CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.18.0] - 2023-06-26
+
+### Added
+- Deploys for Posit Cloud and shinyapps.io now accept the `--visibility` flag.
```

Comment:
We should have an entry here for the change to how we do preflight checking of Shiny app deployability.

Reply:
Would you consider this a fix or a change?

Reply:
I see it as a  change - we're actually removing the preflight check. It might be worth noting that the result will depend on the version of Connect.

Reply:
Do you know which version of Connect the server side check was introduced? If not, I can leave track it down.